crash_similarity.py

```diff
@@ -19,6 +20,22 @@
 pyximport.install()
 
 
+# Checks if the model has been trained in the last 24 hours
+def check_training_time(time_interval):
+    with open('last_trained.txt', 'r') as myfile:
+        data = myfile.read()
+        last_trained_time = timedelta(days=datetime.strptime(data, '%b %d %Y %I:%M%p').day)
+
+    new_training_time = timedelta(days=datetime.today().day)
+
+    return new_training_time - last_trained_time < timedelta(days=1)
```

Comment:
I was using timedeltas only because Kirill told me to do so!
Anyways, now I have made the necessary changes.